test.js

```diff
@@ -1,21 +1,54 @@
 const tape = require('tape')
-const ltest = require('ltest')(tape)
+const rimraf = require('rimraf')
+const LevelTest = require('level-test')
 const listStream = require('list-stream')
 const ttl = require('./')
 const xtend = require('xtend')
-const sublevel = require('level-sublevel')
-const bwSublevel = require('level-sublevel/bytewise')
+const sublevel = require('subleveldown')
 const random = require('slump')
 const bytewise = require('bytewise')
 const bwEncode = bytewise.encode
 
-function test (name, fn, opts) {
-  ltest(name, opts, function (t, _db, createReadStream) {
-    var db
-    var close = _db.close.bind(_db) // unmolested close()
+function ltest (desc, opts, cb) {
+  if (typeof opts === 'function') {
+    cb = opts
+    opts = {}
+  }
+
+  tape(desc, function (t) {
+    var dbName = 'level-test-' + Date.now()
+    var levelup = LevelTest()
+
+    levelup(dbName, opts, function (err, db) {
```

Comment:
Just a thought for later: it would be nice if 1) `level-test` uses `tempy` 2) we skip the rimraf cleanup here and 3) then we no longer need the location argument, because it is optional in `level-test`.

Reply:
Aye, I've thought about that as well.